standalone-network/observers.json

```diff
@@ -1,14 +1,14 @@
 [
   {
     "IsObserver": "y",
-    "ObserverAddress": "zeta1syavy2npfyt9tcncdtsdzf7kny9lh777heefxk",
-    "ZetaClientGranteeAddress": "zeta19wzjdtah4kl2vh77jks68cyy5gpjyurqltys99",
-    "ZetaClientGranteePubKey": "zetapub1addwnpepqggtjvkmj6apcqr6ynyc5edxf2mpf5fxp2d3kwupemxtfwvg6gm7qv79fw0"
+    "ObserverAddress": "zeta13c7p3xrhd6q2rx3h235jpt8pjdwvacyw6twpax",
+    "ZetaClientGranteeAddress": "zeta10up34mvwjhjd9xkq56fwsf0k75vtg287uav69n",
+    "ZetaClientGranteePubKey": "zetapub1addwnpepqtlu7fykuh875xjckz4mn4x0mzc25rrqk5qne7mrwxqmatgllv3nx6lrkdp"
   },
   {
     "IsObserver": "y",
-    "ObserverAddress": "zeta1l7hypmqk2yc334vc6vmdwzp5sdefygj2w5yj50",
-    "ZetaClientGranteeAddress": "zeta1670kf63ny6pltexn5xkkwpk9vqqf6jkvq0ap77",
-    "ZetaClientGranteePubKey": "zetapub1addwnpepq0a2hp5q6sxzf5ry2f39sfhvtrmcpd7z2nfq2z738d8ht6r0pt8hgrqn9tq"
+    "ObserverAddress": "zeta1f203dypqg5jh9hqfx0gfkmmnkdfuat3jr45ep2",
+    "ZetaClientGranteeAddress": "zeta1unzpyll3tmutf0r8sqpxpnj46vtdr59mw8qepx",
+    "ZetaClientGranteePubKey": "zetapub1addwnpepqwy5pmg39regpq0gkggxehmfm8hwmxxw94sch7qzh4smava0szs07kk5045"
```

Comment:
Why do the addresses change?

Reply:
This is for local testing only, I think its because of the change in the default HD path